config/jobs/kubernetes-sigs/cluster-api/templates/cluster-api-presubmits.yaml.tpl

```
@@ -454,3 +454,56 @@ presubmits:
       testgrid-dashboards: sig-cluster-lifecycle-cluster-api{{ if eq $.branch "main" | not -}}{{ TrimPrefix $.branch "release" }}{{- end }}
       testgrid-tab-name: capi-pr-e2e-conformance-ci-latest-{{ ReplaceAll $.branch "." "-" }}
 {{ end -}}
+{{ if eq $.branch "main" }}
+  - name: pull-cluster-api-e2e-latestk8s-{{ ReplaceAll (last $.config.Upgrades).From "." "-" }}-{{ ReplaceAll (last $.config.Upgrades).To "." "-" }}-{{ ReplaceAll $.branch "." "-" }}
```

```suggestion
  - name: pull-cluster-api-e2e-latestk8s-{{ ReplaceAll $.branch "." "-" }}
```
(See related comment in cluster-api-main-presubmits.yaml)

---

done!